blocks/stats/stats.css

```diff
@@ -32,11 +32,20 @@
     margin-bottom: 16px;
     text-align: center;
 
+    h1,h2,h3,h4,h5,h6 {
```

Comment:
Please use the component class as a prefix. Don't write the CSS directly targeting the header tag